### ISSUE TITLE
Truncate overflowing text with an ellipsis (…)

### ISSUE THREAD

**Issue by netzwerg** [fhnw-saav/saav#35]
- [x] In HTML: Truncate all text with ellipsis (...)
- [x] In SVG: Truncate Criteria Labels with ellipsis (...)
- [x] In SVG: Align Sub-Criteria Labels

**Comment by netzwerg** [comment]
It is possible to [embed HTML inside SVG via `foreignobject`](http://stackoverflow.com/questions/39504988/react-svg-html-inside-foreignobject-not-rendered). The HTML is then stylable via CSS, where the truncation/ellipsis behavior is supported out-of-the-box.

However, this approach seems to be [buggy in React](http://stackoverflow.com/questions/39504988/react-svg-html-inside-foreignobject-not-rendered).

**Comment by netzwerg** [comment]
`foreignObject` is broken in scalajs-react (PR).

As soon as it is fixed, the following code will e.g. truncate criteria labels inside SVG:
```
val label =
  <.svg.foreignobject(
    ^.svg.x := layout.getCriteriaAxisX(criteria) - (width / 2f),
    ^.svg.y := layout.boxTopY - layout.padding,
    ^.svg.width := width, ^.svg.height := layout.padding,
    <.div(
      ^.textAlign.center,
      ^.overflow.hidden, ^.textOverflow.ellipsis, ^.whiteSpace.nowrap,
      ^.width := s"${width}px", ^.height := s"${layout.padding}px", ^.minWidth := "0",
      criteria.name
    )
  )
```

**Comment by netzwerg** [comment]
PR was merged and released in scalajs-react 0.11.2

**Comment by netzwerg** [comment]
Done
